app/frontend/wallet/shelley/shelley-address-provider.ts

```diff
@@ -26,7 +26,8 @@ const shelleyStakeAccountPath = (account: number) => {
 }
 
 const _validateScheme = (cryptoProvider) => {
```

Comment:
I would rather remove it completely - it's an "internal" function after all and I really hope that Shelley will learn from Byron "mistakes" and won't need to have multiple derivation schemes